packages/cli/src/tools/npm.ts

```diff
@@ -129,7 +127,7 @@ const minorVersion = (version: string) => {
 export async function getTemplateVersion(
   reactNativeVersion: string,
 ): Promise<TemplateVersion | undefined> {
-  const json = await fetch(TEMPLATE_VERSIONS_URL).then(
+  const json = await fetch(getNpmRegistryUrl() + '@react-native-community/template').then(
```

Comment:
I'd prefer leveraging here `new URL()` to make sure URL is always in a correct format, e.g. when `getNpmRegistryUrl()`'d return value without `/` at the end, previous would've fail.

```suggestion
  const json = await fetch(new URL('@react-native-community/template', getNpmRegistryUrl())).then(
```